next-app/src/lib/error/index.ts

```diff
@@ -1,30 +1,31 @@
-import { error as serverError } from '@sveltejs/kit'
-import { browser } from '$app/environment'
-import { writable } from 'svelte/store'
+import { error as serverError } from '@sveltejs/kit';
+import { browser } from '$app/environment';
+import { writable } from 'svelte/store';
 
-export const error = writable()
+export const error = writable();
 
 export function throwError(message = '', code = 0) {
-	if (browser) throw set({ code, message })
+  if (browser) throw set({ code, message });
```

Comment:
are these tabs or spaces?  I personally prefer tabs, maybe we can discuss this rule.

Reply:
You'd prefer tabs just in `next-app`?

As I still haven't touched application code (only testing code) I'm actually not even sure what `next-app/src` vs `/src` are 🤔, but `/src` seems to have been using 2-space indentation.

Reply:
we can talk about it, I do prefer tabs and find 2-space indent to require further interpretation of context, i.e., it takes me longer to confirm in my mind that the line is actually indented.